packages/calling-stateful-client/src/CallContext.ts

```diff
@@ -706,7 +706,6 @@ export class CallContext {
     if (
       latestCaption &&
       latestCaption.resultType !== 'Final' &&
```

Comment:
```
latestCaption.resultType !== 'Final'
```

Is it this line should be removed? Why does the time comparing need to be removed, I can feel this would cause lots of issues if we remove the time compare